embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/logs/CompositeLogService.kt

```diff
@@ -24,7 +24,7 @@ internal class CompositeLogService(
 ) : LogMessageService {
 
     private val useV2LogService: Boolean
-        get() = configService.sessionBehavior.useV2Payload()
+        get() = configService.oTelBehavior.isBetaEnabled()
```

Comment:
Have we tested this in production? If we are confident that this should mostly work, we should put this is stable. In fact, this is the one that I would most like to be in stable.

Reply:
Let's leave it in Beta for now, as we need to fix some tests once we enable this.

Reply:
Ah got it. Make sense.